scripts/jenkins/ardana/ansible/roles/ardana_deploy/defaults/main.yml

```diff
@@ -23,9 +23,6 @@ ardana_openstack_playbooks:
   - "ready-deployment.yml"
 
 ardana_scratch_playbooks:
-  - play: "ardana-ssh-keyscan.yml"
```

Comment:
is wipe_disks doing the keyscan now or was this extraneous?

Reply:
The fact that this was being run is what allowed the older version of wipe_disks.yml to avoid hitting the host key issue. Now that wipe_disks.yml is running the playbook, we can remove running the playbook.